carbonmark-api/src/routes/users/[walletOrHandle]/get.ts

```diff
@@ -82,9 +83,13 @@ const handler = (fastify: FastifyInstance) =>
           };
           return {
             ...a,
-            amount: utils.formatUnits(a.amount || "0", 18),
+            amount: a.project.key.startsWith("ICR")
```

Comment:
Also this logic that compute the price format should be in the `fetchCarbonProject` method to avoid redundancy

Reply:
In the current fetchCMSProject? There shouldn't be any overlap as that is all project info and the user route has the activities and assets. I might be missing something though?

Reply:
I agree here but I don't see a clean solution.

Having registry specific ternaries and logic spread around the codebase is not great. 

Can we either 
a.) Move this further up the chain or 
b.) Extract the decimals for each registry in a constant somewhere and reference it when calling `format`?

Reply:
I put this issue in this ticket: https://github.com/KlimaDAO/klimadao/issues/2136